AudioReminderCore/FIlePathHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AudioReminderCore
+{
+    /// <summary>
+    /// Path related helper methods
+    /// </summary>
+    public class FilePathHelper
```

Comment:
Hi @super-ivanka ,

This is a very useful utility method. Thanks!

Reply:
Poy